dom-renderers/DOMRenderer.js

```diff
@@ -117,6 +117,29 @@ DOMRenderer.prototype.subscribe = function subscribe(type, preventDefault) {
 };
 
 /**
+ * Removes an EventListener of given type from the element on which it was
+ * registered.
+ *
+ * @method
+ *
+ * @param {String} type DOM event type (e.g. click, mouseover).
+ *
+ * @return {undefined} undefined
+ */
+DOMRenderer.prototype.unsubscribe = function unsubscribe(type) {
+    this._assertTargetLoaded();
+    this._target.subscribe[type] = false;
+
+    if (
+        this._target.listeners[type] || this._root.listeners[type]
+    ) {
+        var target = eventMap[type][1] ? this._root : this._target;
+        target.listeners[type] = null;
+        target.element.removeEventListener(type, this._boundTriggerEvent);
```

Comment:
@alexanderGugel I think there is an issue where if you don't remove the listener from the DOM you will intercept the click from any other DOMElement further back in Z.  This would mean we need a full remove vs just not listening application side.